inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_boot_tests.cpp

```diff
@@ -61,19 +61,11 @@ TEST_P(MYRIADBoot, DISABLED_ConnectToAlreadyBootedDevice) {
     bootOneDevice();
     ASSERT_EQ(getAmountOfBootedDevices(), 1);
     {
-        InferenceEnginePluginPtr plugin(make_plugin_name(GetParam().pluginName));
-        CNNNetReader reader;
-        reader.ReadNetwork(GetParam().model_xml_str.data(), GetParam().model_xml_str.length());
-
-        CNNNetwork network = reader.getNetwork();
-        ExecutableNetwork ret;
-
-        sts = plugin->LoadNetwork(ret, network, {
-                {KEY_LOG_LEVEL, LOG_DEBUG},
-                {KEY_VPU_MYRIAD_WATCHDOG, NO},
-        }, &response);
-
-        ASSERT_NE(StatusCode::OK, sts) << response.msg;
+        Core ie;
+        CNNNetwork network = ie.ReadNetwork(GetParam().model_xml_str, Blob::CPtr());
```

Comment:
Could you please explain a bit what is use case here? Do we leave weights empty? Is it more convenient to pass `nullptr` as second argument? Is it possible not pass it at all?

Reply:
> Is it possible not pass it at all?

No, in this case the method with reading from file will be called

Reply:
Looks like we need something like `ReadModelOnly` method :)